docs/recipes/README.md

```diff
@@ -2,63 +2,19 @@
 
 ## Throttling
 
-You can throttle a sequence of dispatched actions by putting a delay inside a watcher Saga. For example, suppose the UI fires an `INPUT_CHANGED` action while the user is typing in a text field.
+You can throttle a sequence of dispatched actions by using a handy built-in `throttle` helper. For example, suppose the UI fires an `INPUT_CHANGED` action while the user is typing in a text field.
 
 ```javascript
-const delay = (ms) => new Promise(resolve => setTimeout(resolve, ms))
-
 function* handleInput(input) {
   // ...
 }
 
 function* watchInput() {
-  while (true) {
-    const { input } = yield take('INPUT_CHANGED')
-    yield fork(handleInput, input)
-    // throttle by 500ms
-    yield call(delay, 500)
-  }
+  yield throttle(500, 'INPUT_CHANGED', handleInput)
```

Comment:
As a user it would be nice to have `throttle` imported in this example so that I know where to find it from.